### ISSUE TITLE
[BUG] I can't export issues to github repository

### ISSUE THREAD

**Issue by DecampsRenan** [BearStudio/start-repo#24]
## What's the pb ?

I tried to export "Website" issues to a **private repo** on github, and it seems there is something wrong with TRPC

![CleanShot 2022-03-15 at 15 06 26](https://user-images.githubusercontent.com/7521065/158395995-542ef45a-6eb8-4d4f-a662-74acf2483a1b.png)

![CleanShot 2022-03-15 at 15 09 16](https://user-images.githubusercontent.com/7521065/158396184-ec5b91ec-ecb4-429b-a051-720aee658157.png)

**Comment by yoannfleurydev** [comment]
(Prod only)

**Comment by yoannfleurydev** [comment]
OK, so the cookie used in production didn't used the same name as [the one in localhost](https://github.com/BearStudio/start-repo/commit/036f8575727fb7879b22efc771be313c39880ac1#diff-2499c95c7a5e0d9c0c36c9a346c3325a719fc789518619dd1ea5889935d85bbcL149) (the production one was prefixed with some `__Secure` text). So, for better user accessToken management, I deleted the code that was using the cookie thing and used the [`getSession({ ctx })`](https://next-auth.js.org/getting-started/client#getsession) method that I supercharged with data using the [`async session`](https://next-auth.js.org/configuration/callbacks#session-callback) callback. Of course, it needed some TypeScript adaptation that I did.
